swarm/src/behaviour.rs

```diff
@@ -621,6 +621,50 @@ where
     }
 }
 
+impl<TInEventOld, TOutEvent, THandlerOld> NetworkBehaviourAction<TOutEvent, THandlerOld>
+where
+    THandlerOld: IntoProtocolsHandler,
+    <THandlerOld as IntoProtocolsHandler>::Handler: ProtocolsHandler<InEvent = TInEventOld>,
+{
+    /// Map the handler and hanlder event
+    pub fn map_handler_and_in_event<THandlerNew, TInEventNew>(
```

Comment:
```suggestion
    pub fn map_handler_and_in<THandlerNew, TInEventNew>(
```

For the sake of consistency with `map_out`. What do you think? I am as well fine with renaming `map_out` to `map_out_event`.